src/doc/src/reference/external-tools.md

```diff
@@ -214,8 +214,8 @@ may be found in [the chapter on build scripts](build-scripts.md).
         ["SOME_KEY", "some value"],
         ["ANOTHER_KEY", "another value"]
     ],
-    /* A path which is used as a value of `OUT_DIR` environmental variable
-       when compiling current package.
+    /* An absolute path which is used as a value of `OUT_DIR` environmental
+       variable when compiling current package.
     */
     "out_dir": "/some/path/in/target/dir"
```

Comment:
Should this example line be changed to look more like an absolute path?

Reply:
It starts with a `/` already, so that should be fine I think.